### DISCUSSION TITLE
Q: 评论产生的跨域问题，弹出Unable to fetch提示框（偶发） || Q: The Unable to fetch prompt box pops up due to cross-domain problems generated by comments (occasionally)

### DISCUSSION THREAD

**Discussion by uuanqin** [walinejs/waline#2022] [Q&A] [answered]
### Describe the bug | 问题描述

回复他人评论时，会弹出Unable to fetch提示，但是评论是成功的。

控制台截图：
![QQ截图20230712153400](https://github.com/walinejs/waline/assets/61202052/0e6ac6a7-8871-4883-94ea-969c61b31d79)

这种情况偶然发生，一般翻墙后又不会发生，我也不懂什么情况。

虽然评论功能正常，但是这个提示框确实挺影响体验的。

绑定的域名是：pl.uuanqin.top

### Website URL | 问题网站

https://www.uuanqin.top/building.html

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

When replying to other people's comments, the Unable to fetch prompt will pop up, but the comments are successful.

Console screenshot:
![QQ screenshot 20230712153400](https://github.com/walinejs/waline/assets/61202052/0e6ac6a7-8871-4883-94ea-969c61b31d79)

This kind of situation happens occasionally, and it usually doesn't happen after overcoming the wall. I don't know what the situation is.

Although the comment function is normal, this prompt box really affects the experience.

The bound domain name is: pl.uuanqin.top

### Website URL | Problem Website

https://www.uuanqin.top/building.html

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is your comment data store?

LeanCloud (https://leancloud.app)

**Comment by lizheming** [comment]
看起来是网络问题导致接口失败了，我理解是符合预期的

---
It seems that the network problem caused the interface to fail. I understand that it is in line with expectations

**Reply by uuanqin** [reply]
对于弹出“Fail To Fetch”的情况进行了定位，与翻墙与否无关，只在 **登录用户评论非登录用户** 、**非登录用户评论登录用户** 时**必会**出现，非登录用户评论非登录用户时不会出现。可能是一个小小的bug吧...

![image](https://github.com/walinejs/waline/assets/61202052/cacac820-d4ed-476b-be31-aa0153dc5411)



---
For the pop-up "Fail To Fetch" situation, it has nothing to do with whether to bypass the wall or not. It will only happen when **logged-in users comment on non-logged-in users** and **non-logged-in users comment on logged-in users** Appears, and does not appear when non-logged-in users comment on non-logged-in users. Might be a small bug...

![image](https://github.com/walinejs/waline/assets/61202052/cacac820-d4ed-476b-be31-aa0153dc5411)

**Reply by lizheming** [reply, marked as answer]
去你网站试了一下，和登录未登录没什么关系，评论是存储成功了，但我看是 Vercel 那边接口 timeout 了，应该是你配置的邮箱服务有点问题，可以先把评论回复通知的邮箱服务关下试试。

---
I went to your website and tried it. It has nothing to do with logging in or not. The comment is stored successfully, but I think it is the interface timeout on the Vercel side. It should be that there is something wrong with the email service you configured. You can reply the comment to notify the email service first. Try it off.
